php/commands/role.php

```diff
@@ -258,6 +262,7 @@ public function reset( $args, $assoc_args ) {
 		global $wp_roles;
 		$all_roles = array_keys( $wp_roles->roles );
 		$preserve_args = $args;
+		$this->dry_run         = \WP_CLI\Utils\get_flag_value( $assoc_args, 'dry-run' );
```

Comment:
No, we don't need dry-run as class variable as we are not using it in any other method. :+1: